cmd/dep/status_test.go

```diff
@@ -555,3 +556,102 @@ func TestValidateFlags(t *testing.T) {
 		})
 	}
 }
+
+func TestStatusMissing(t *testing.T) {
+
+	cases := []struct {
+		name       string
+		lock       dep.Lock
+		wantStatus string
+		wantErr    bool
+	}{
+		{
+			name: "no missing dependencies",
+			lock: dep.Lock{
+				P: []gps.LockedProject{
+					gps.NewLockedProject(
+						gps.ProjectIdentifier{ProjectRoot: gps.ProjectRoot("github.com/boltdb/bolt")},
+						gps.NewVersion("v1.0.0"),
+						[]string{"."},
+					),
+					gps.NewLockedProject(
+						gps.ProjectIdentifier{ProjectRoot: gps.ProjectRoot("github.com/sdboyer/dep-test")},
+						gps.NewVersion("v1.0.0"),
+						[]string{"."},
+					),
+					gps.NewLockedProject(
+						gps.ProjectIdentifier{ProjectRoot: gps.ProjectRoot("github.com/sdboyer/deptestdos")},
+						gps.NewVersion("v1.0.0"),
+						[]string{"."},
+					),
+				},
+			},
+			wantStatus: "No missing dependencies found.\n",
+		},
+		{
+			name: "two missing dependencies",
+			lock: dep.Lock{
+				P: []gps.LockedProject{
+					gps.NewLockedProject(
+						gps.ProjectIdentifier{ProjectRoot: gps.ProjectRoot("github.com/sdboyer/deptestdos")},
+						gps.NewVersion("v1.0.0"),
+						[]string{"."},
+					),
+				},
+			},
+			wantStatus: "Missing dependencies (not present in lock):\n" +
+				"  github.com/boltdb/bolt\n" +
+				"  github.com/sdboyer/dep-test\n",
```

Comment:
You can use back tick (`) as well if you would like to avoid appending with + and \n.